lms/djangoapps/courseware/date_summary.py

```diff
@@ -167,9 +134,7 @@ def date(self):
 
     @property
     def title(self):
-        return _(u'Today is {date}').format(
-            date=self.date.astimezone(self.time_zone).strftime(self.date_format.encode('utf-8')).decode('utf-8')
-        )
+        return 'Today is {date}'
```

Comment:
Why is the call to ugettext removed from here?  This string will no longer get translated correctly.